index.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+
+    <title>Intuit Accounts - Sign Up</title>
+
+    <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+
+    <header>
+        <div class="header-section">
+            <div class="signin-container">
+                <div class="signin-text">
+                    <p class="p-signin">Already have an Intuit account?</p>
+                    <a href="#">I forgot my User ID or Password</a>
+                </div>
+                <div>
+                    <button class="signin-button">
+                        Sign In
+                    </button>
+                </div>
+            </div>
+            <div class="logo-container center-div">
+                <a href="#"><img src="images/intuit-logo.png" alt="Intuit Logo"></a>
+            </div>
+            <div class="services-container center-div">
+                <a href="#" class="serviceIcon"><img src="images/turboIcon.png" alt="Turbotax Logo"></a>
+                <a href="#" class="serviceIcon"><img src="images/quickIcon.png" alt="Quickbooks Logo"></a>
+                <a href="#" class="serviceIcon"><img src="images/mintIcon.png" alt="Mint Logo"></a>
```

Comment:
`AWESOME`

Nice job, :clap: :star2:  adding an `alt` attribute to the `<img>` element, this is a very important accessibility attribute.
`Examples of scenarios where users benefit from text alternatives for images` are listed below and could also be found [here](https://www.w3.org/TR/html51/semantics-embedded-content.html#alt-text)

- They have a very slow connection and are browsing with images disabled.
- They have a vision impairment and use text to speech software.
- They have a cognitive impairment and use text to speech software.
- They are using a text-only browser.
- They are listening to the page being read out by a voice Web browser.
- They have images disabled to save on download costs.
- They have problems loading images or the source of an image is wrong.


##### SOURCE: [W3C Recommendation (Embedded Content)](https://www.w3.org/TR/html51/semantics-embedded-content.html#alt-text)